File: src/main/kotlin/IWannaBeTheGuy.kt
package problems

/* Problem Link: https://codeforces.com/problemset/problem/469/A */

fun main() {
    val n = readln().toInt()
    val levels = mutableSetOf<String>()
    val x = readln().split(" ")
    if (x.size > 1) levels.addAll(x.drop(1))

    val y = readln().split(" ")
    if (y.size > 1) levels.addAll(y.drop(1))

    print(if (levels.size == n) "I become the guy." else "Oh, my keyboard!")
}

File: Codeforces/C++/problemset/I Wanna Be the Guy.cpp
// Problem Link: https://codeforces.com/problemset/problem/469/A

#include <iostream>
#include <set>
using namespace std;
typedef long long ll;
int main()
{
    ll n,p,q,a;
    set <ll> all;
    cin >> n;
    cin >> p;
    for(ll i = 0; i < p; i++)
    {
        cin >> a;
        all.insert(a);
    }
    cin >> q;
    for(ll i = 0; i < q; i++)
    {
        cin >> a;
        all.insert(a);
    }
    (all.size() == n) ? cout << "I become the guy." : cout << "Oh, my keyboard!";
    return 0;
}
